libs/dotnet-sdk/Registration/RegisterFeatureBoard.cs

```diff
@@ -11,32 +12,45 @@ namespace FeatureBoard.DotnetSdk.Registration;
 public static class RegisterFeatureBoard
 {
   private static UpdateStrategy _updateStrategy = UpdateStrategy.None;
-  public static FeatureBoardBuilder AddFeatureBoard<TFeatures, TAudienceProvider>(this IServiceCollection services) where TFeatures : class, IFeatures where TAudienceProvider : class, IAudienceProvider
+
+  private static EntityTagHeaderValue? lastETag = null;
+
+  public static FeatureBoardBuilder AddFeatureBoard<TFeatures>(this IServiceCollection services, IConfigurationRoot configuration) where TFeatures : class, IFeatures
   {
-    var configuration = services.BuildServiceProvider().GetRequiredService<IConfiguration>();
```

Comment:
Out of curiosity why pass this in rather than resolve like this?

Reply:
No _major_ reason, just that it saves the service provider from being built twice. It is a micro-optimation that my inner pedant couldn't control himself over... 😉

Reply:
Happy to revert if there are reasons not to do it that I've missed btw.